blocks/generators.js

```diff
@@ -1,9 +1,86 @@
-Blockly.Python['servo'] = function(block) {
-  Blockly.Python.definitions_['from_machine_import_pin'] = 'from machine import Pin';
-  Blockly.Python.definitions_['from_machine_import_pwm'] = 'from machine import PWM';
+Blockly.Python["pin_motor_right_stop"] = function (block) {
+  Blockly.Python.definitions_["from_machine_import_pin"] =
+    "from machine import Pin";
+  var pin1 = 14,
+    pin2 = 18;
+  var code = `Pin(${pin1}, Pin.OUT).value(${false})\nPin(${pin2}, Pin.OUT).value(${false})\n`;
+  return code;
+};
+
+Blockly.Python["pin_motor_right_forward"] = function (block) {
+  Blockly.Python.definitions_["from_machine_import_pin"] =
+    "from machine import Pin";
+  var pin1 = 14,
+    pin2 = 18;
+  var code = `Pin(${pin1}, Pin.OUT).value(${true})\nPin(${pin2}, Pin.OUT).value(${false})\n`;
```

Comment:
why you didn't put the two pins of the left motor to zero as I have explained to you?